eng/pipelines/templates/steps/test-nightly-python38.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+  AdditionalTestArgs: ''
+  TestMarkArgument: ''
+  EnvVars: {}
+  ServiceDirectory: ''
+  PythonVersion: ''
+  OSName: ''
+  BeforeTestSteps: []
+  CoverageArg: ''
+  BuildTargetingString: 'azure-*'
+  ToxTestEnv: ""
+  RunCoverage: ne(variables['CoverageArg'], '--disablecov')
```

Comment:
I don't see this being used below. am I wrong?

Reply:
Thanks for pointing it out. I have removed it and committed new change.